### PR TITLE
Queries for Sql practice tasks were added, please review

### PR DESCRIPTION
**Pull request by annaoliinyk**

### CODE REVIEW

File: SQL_practice.sql [hunk 1]
```diff
@@ -0,0 +1,87 @@
+--1
+SELECT * FROM patients
+where province_id is "ON";
+
+
+
+-- 2
+select Pr.province_name, count(Pa.patient_id) as 'Total count of patients'
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Please do not use spaces as a separator for aliases. Also they should be shorter, e.g. total_count

File: SQL_practice.sql [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+-- 3
+select distinct Pr.province_name, Pa.city
+from patients Pa
+ join provinces Pr
+ on Pa.province_id = Pr.province_id
+order by Pr.province_name;
+
+
+-- 4
+select A.vendor_name as 'First vendor name',
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Same here. Please do not use spaces as a separator for aliases.

File: SQL_practice.sql [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  case
+	   when B.vendor_name = A.vendor_name
+		    then ''
+			   else B.vendor_name
+				  end as 'Second vendor name',
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Same here. Please do not use spaces as a separator for aliases.

File: SQL_practice.sql [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+ join provinces Pr on V.province_id = Pr.province_id
+group by V.city
+having count(V.city) is 1;
+
+-- with count of unique cities for vendors
+select P.province_name, 
+  ( select count(*) from
+	     (select province_id, count(city) from vendors group by city having count(city) is 1)
+	                     group by province_id
+			  ) as 'Number on unique cities in province'
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Same here. Please do not use spaces as a separator for aliases.

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change:
- all queries should be in the separate files. Single responsibility principle.
- style should be the same for all queries (read sql conventions on the materials column)
- please re-check task 4, there are 8 unique vendors in each city and your query returns 7 and two vendor’s names
- also please re-check 6, the query gives me no result and I do not get what’s going on with queries “for vendors”

**Review by annaoliinyk — COMMENTED** [review]
Fixed as requested

File: 1.sql [hunk 5]
```diff
@@ -0,0 +1,3 @@
+SELECT *
+ FROM patients
+where province_id is "ON";
```
Comment (AnaKovaGH) [inline 1.sql]: Change reserved words to UPPER case and move * to the new line

File: SQL_practice.sql [hunk 6]
```diff
@@ -55,13 +54,13 @@ select P.province_name,
   ( select count(*) from
 	     (select province_id, count(city) from vendors group by city having count(city) is 1)
 	                     group by province_id
-			  ) as 'Number on unique cities in province'
+			  ) as 'n_unique_cities'
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Please try to avoid only one letter, e.g. not n_unique_cities, but number_unique_cities

**Comment by AnaKovaGH** [comment]
Please change, what we have discussed during the call

File: SQL_practice.sql [hunk 7]
```diff
@@ -0,0 +1,88 @@
+--1
```
Comment (AnaKovaGH) [inline SQL_practice.sql]: Please change 1 on the description of the task

**Review by AnaKovaGH — APPROVED** [review]
LGTM!

**Comment by AnaKovaGH** [comment]
Please fix the conflicts and merge to master.

**Pull request merged**